content/applications/inventory_and_mrp/inventory/shipping/operation/label_type.rst

```diff
@@ -0,0 +1,73 @@
+=====================================
+How to change the shipping label size
+=====================================
+
+Overview
+========
+
+With Odoo, there are a variety of different types of shipping  labels that can be selected for
+delivery orders. Depending on the types of shipping packages used, different label sizes may be more
+appropriate, and can be easily configured in Odoo.
+
+Configuration
+=============
+
+In the :guilabel:`Inventory` module, go to :menuselection:`Configuration --> Delivery --> Shipping
+Methods.` Choose a delivery method and then click :guilabel:`Edit`. For the following example,
+*FedEx International* will be used. Choose a delivery method and then click on :guilabel:`Edit`.
+
+.. image:: label_type/shipping-options.png
+   :align: center
+   :alt: Different shipping methods.
+
+In the :guilabel:`Configuration` tab, under :guilabel:`Label Type`, choose one of the label types
+available. The availability will vary depending on the carrier. Save the changes.
+
+.. image:: label_type/label-type-dropdown.png
+   :align: center
+   :alt: Select a label type.
+
+When a *sales order* with the corresponding shipping company is confirmed, and the *delivery order*
+is validated, the shipping label will be automatically created as a PDF and appear in the
+:guilabel:`Chatter`.
+
+Create a Sales Order
+====================
+
+In the :guilabel:`Sales` application, create a quotation for an international customer with a
+storable product. Click on :guilabel:`Create`, and select an international customer for the quote.
```

Comment:
Delete "on" and the "," in: Click on :guilabel:`Create`,

Instead: Click :guilabel:`Create`